kits/fablens/prompts/fabric-material-analyzer_llmnode-449_system_0.md

```
@@ -0,0 +1,29 @@
+- Keep rayon and viscose as distinct if both appear
+Output format:
+{
+"materials": ["rayon", "viscose"]
+}
+If none found:
+{
+"materials": [],
+"note": "No material information found on page"
+}
+Return ONLY valid JSON.
+No explanation.
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)